lib/patir/command.rb

```diff
@@ -134,7 +134,11 @@ def run context=nil
               @error<<"Command timed out after #{@timeout}s"
               exited=true
               exitstatus=23
-              Process.kill 9,cid
+              begin
+                Process.kill 9,cid
+              rescue => ex
+                @error<<"Failure to kill timeout child process #{cid}: #{ex.message}"

```

Comment:
Make this "Failed to kill child process #{cid} after timeout", better phrasing

Reply:
Done :+1: